python/lib/dependabot/python/update_checker/pipenv_version_resolver.rb

```diff
@@ -151,9 +148,18 @@ def handle_pipenv_errors(error)
             raise DependencyFileNotResolvable, msg
           end
 
-          if error.message.include?("Could not find a version") ||
-             error.message.include?("is not a python version")
-            check_original_requirements_resolvable
+          if error.message.match?(PIPENV_RANGE_WARNING)
```

Comment:
When using a python range, we already know updates are not possible so we don't need to retry the update